inc/class-s3-uploads-stream-wrapper.php

```diff
@@ -17,7 +17,35 @@ public static function register( Aws\S3\S3Client $client)
 		static::$client = $client;
 	}
 
+	// Override
+	public function stream_flush() {
+
+		// Add some HTTP headers which can be defined in wp-config.php
+
+		// Expires:
+		if ( defined( 'S3_UPLOADS_EXPIRES' ) ) {
+			$this->params[ 'Expires' ] = S3_UPLOADS_EXPIRES;
+		}
+
+		// Cache-Control:
+		if ( defined( 'S3_UPLOADS_CACHE_CONTROL' ) ) {
```

Comment:
Don't think we can do this.  The class is registered using stream_wrapper_register() in S3_Uploads_Stream_Wrapper::register()